src/server/array_func.c

```diff
@@ -806,15 +808,19 @@ create_subjob(job *parent, char *newjid, int *rc)
 	 * non-saved items before ji_qs.
 	 */
 
-	subj = job_alloc();
+	if (NULL == (subj = job_alloc())) {
```

Comment:
Please keep the value at RHS. I know it will avoid accidental assignment. But please follow what the rest of the PBS does. Also value at RHS is more easy to read.

Reply:
addressed in commit named rc1 https://github.com/PBSPro/pbspro/commit/b2f22afcadb50bc872cb198d65b51190511aaf45